R/zzz.R

```diff
@@ -11,15 +11,5 @@ get_train_template_id <- function() {
   latest <- max(versions)
   id <- CIVIS_ML_TEMPLATE_IDS[CIVIS_ML_TEMPLATE_IDS$version == latest &
                               CIVIS_ML_TEMPLATE_IDS$name == "train", "id"]
-
-  # only check update if API key is set
-  if (!inherits(try(api_key(), silent = TRUE), "try-error")) {
-    if (length(scripts_list_custom(id)) == 0) {
-      # use the previous version; assume that only the latest version is in internal release
```

Comment:
I don't think we should remove this check. External users won't have access to templates until they get permissions on Wednesday, which means there will be a period where 2.1 is also in "internal release" by default. In fact, we'll have some clients who are on 1.1, some who are beta users on 2.0, and internal users who are on 2.1. So in fact, I think we should make this logic more general, and check against all available versions of CivisML until we find one we can use (e.g, check for 2.1, if fail, check for 2.0, if fail, check for 1.1, etc.). This logic will be useful for any future releases as well, and it's how the python client is handling it now.